Dockerfile

```diff
@@ -31,7 +31,7 @@ COPY --from=builder /go/bin/gardener-extension-shoot-rsyslog-relp-admission /gar
 ENTRYPOINT ["/gardener-extension-shoot-rsyslog-relp-admission"]
 
 ############# rsyslog-relp-echo-server
-FROM alpine:3.18.4 AS shoot-rsyslog-relp-echo-server
+FROM europe-docker.pkg.dev/gardener-project/public/3rd/alpine:3.18.4 AS shoot-rsyslog-relp-echo-server
```

Comment:
I think we don't do this for Dockerfiles. The whole idea of copying images to AR is to deal with dockerhhub rate limits and with registries that don't support image pulls over IPv6.
IMO, you could revert here.